README.md

```
@@ -1,20 +1,53 @@
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The following steps can be used to run only the tests relevant to this project.
 
-Things you may want to cover:
+In spec_helper.rb:
 
-* Ruby version - 3.4.5
+1. Add the following line to the require section:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)